dask_expr/collection.py

```diff
@@ -18,7 +18,7 @@
 from tlz import first
 
 from dask_expr import expr
-from dask_expr.expr import RenameFrame, no_default
+from dask_expr.expr import ExplodeFrame, ExplodeSeries, RenameFrame, no_default
```

Comment:
It feels like we started to use the convention of `expr.ExplodeFrame` in this file to avoid the need to import everything by name here. Can we continue using that convention for `ExplodeFrame` and `ExplodeSeries`?

Reply:
Oh yes, good point. Changed